nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -341,6 +341,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 <!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
+- `i686-linux` NixOS tests requiring a system image (i.e. boot-related, etc.) are broken until [LKL](https://lkl.github.io/) is fixed on i686-linux or someone introduces a slowpath in `make-disk-image` for creating images without `cptofs` (fully in VM for example) which would only depend on QEMU then.
```

Comment:
Not sure breakages warrant release notes entries.

Reply:
This is fairly big breakage for i686 users in NixOS, I'd say.